app/presenters/notices/setup/notifications.presenter.js

```diff
@@ -35,6 +35,11 @@ const MESSAGE_REFS = {
       'Single use': 'returns_invitation_licence_holder_letter'
     }
   },
+  failedInvitations: {
```

Comment:
I'd add some unit tests for this new notice type as we appear to have them for all the others.